locale/templates/settings.ftl

```diff
@@ -527,6 +535,8 @@ se-make-primary = Make primary
 se-default-content = Access your account if you can’t log in to your primary email.
 se-content-note = Note: a secondary email won’t restore your information — you’ll
   need a <a>recovery key</a> for that.
+# Default value for the secondary email
+se-secondary-email-none = None
 
 ##
```

Comment:
Remove this if not used.

Reply:
This is used @ https://github.com/mozilla/fxa/blob/main/packages/fxa-settings/src/components/UnitRowSecondaryEmail/index.tsx#L139